client/main.go

```diff
@@ -71,12 +71,19 @@ func main() {
 			Name:  "create",
 			Usage: "Create ToDD object (group, testrun, etc.)",
 			Action: func(c *cli.Context) {
+
+				yamlFileName := ""
+				// If a filename is passed, use that. Otherwise, pass an empty string
+				if len(c.Args()) >= 1 {
+					yamlFileName = c.Args()[0]
+				}
+
 				clientapi.Create(
 					map[string]string{
 						"host": host,
 						"port": port,
 					},
-					c.Args()[0],
```

Comment:
You could use `c.Args().Get(0)` or `c.Args().First()`, both return an empty string if there are no arguments.